EventProc/python/overlay.py

```diff
@@ -0,0 +1,38 @@
+"""Configuration for overlay """
+
+
+
+from LDMX.Framework import ldmxcfg
+
+class OverlayProducer(ldmxcfg.Producer) :
+    """Configuration for pileup overlay
+
+        Sets all parameters to reasonable defaults.
+
+    Examples
+    --------
+        from LDMX.EventProc.overlay import OverlayProducer
+        p.sequence.append( OverlayProducer() )
+    """
+
+    def __init__(self,name = 'overlay') :
+        super().__init__(name,'ldmx::OverlayProducer','EventProc')
+
+
+        self.overlayFileName = "ldmx_upstreamMultiElectron_events.root"
+        self.overlayTreeName = "LDMX_Events"
+        self.numberOverlaidInteractions = 1.
+        self.timeSpread = 0.
+        self.doPoisson = 0
+        self.overlayProcessName = "inclusive"
```

Comment:
Since you are just reading from a file. This isn't super necessary to be configurable. I would keep it as a Cpp member variable, but remove it here to clean up the python config.